src/components/Layout/Desktop.jsx

```diff
@@ -25,21 +27,14 @@ export default function Layout() {
         <Map className={useDarkTheme ? "dark-theme" : ""} />
       </Layer>
       <Layer>
-        <Block position={'fixed'} bottom={0} left={0} width={['100%', '100%', 'auto']} margin={['0', '0', '20px']}>
-          <Block
```

Comment:
This part of code (entire <Block> with override styles should be included as well :P